packages/core/src/sdk/session/index.ts

```diff
@@ -50,6 +50,16 @@ export const mutation = gql(`
 `)
 
 export const validateSession = async (session: Session) => {
+  if (storeConfig.deliveryPromise?.enabled && !session.postalCode) {
```

Comment:
What should happen if `deliveryPromise?. enabled` is false and the `postalCode` is set in the session? I think in this situation, the `postalCode` is set in the initial `createSessionStore(storeConfig.session)`. I'm not sure if that is the expected behavior.

Reply:
Yes, in the initial one the session would be created with the `postalCode` value set in the default config file. I believe it makes sense, because it was the store's choice to put that in the default file. 
The difference is that it would work only for the first session creation. If the session was already created previously or if the `postalCode` in the session becomes empty for whichever reason, if the `enabled` is false then it won't be updated to use the default `postalCode`. This necessary update will happen only when `enabled` is true because this behavior is related to the Delivery Promise feature.

Reply:
Nice! Agreed